google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugProcessStateCollector.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Why do we need this new class? From its name, its functionality sounds similar to CloudDebugGlobalPoller.